src/renderers/dom/shared/hooks/ReactDOMUnknownPropertyHook.js

```diff
@@ -67,6 +67,11 @@ if (__DEV__) {
     if (EventPluginRegistry.registrationNameModules.hasOwnProperty(name)) {
       return true;
     }
+    if (EventPluginRegistry.plugins.length === 0 && name.indexOf('on') === 0) {
```

Comment:
FWIW I considered this but thought it wasn't worth the tradeoff- (regex is much slower)- given that the DOM render would warn about onions when the client took over.